src/components/Movie.js

```diff
@@ -0,0 +1,85 @@
+
+//url = `http://www.omdbapi.com/?i=${id}&plot=full&apikey=d2807699`
+
+import React from 'react';
+
+import cx from 'classnames';
+
+
+class Movie extends React.Component{
+  constructor(){
+    super();
+    this.state ={
+      movieDetails:{},
+      showDetails:false,
+
+    }
+
+    this.handleClick = this.handleClick.bind(this);
+    this.fetchMovieDetails = this.fetchMovieDetails.bind(this);
+    this.handleFavClick = this.handleFavClick.bind(this);
+
+  }
+
+  handleClick(){
+    this.fetchMovieDetails();
+    this.setState({
+      showDetails:!this.state.showDetails
+    })
+  }
+
+  handleFavClick(){
+    this.props.receiveFavClick(this.props.id)
+    
+  }
+
+  fetchMovieDetails(){
+    //console.log(this.state.keyWord)
+    let detailsUrl=`http://www.omdbapi.com/?i=${this.props.id}&plot=full&apikey=d2807699`
+    fetch(detailsUrl)
+    .then(response => response.json())
+    .then(body => {
+      //console.log(body)
+      this.setState({
+        movieDetails:body
+      })
+
+    })
+
+  }
+
+
+  render(){
+    let check={fav_on:false};
+
+    this.props.favList.find(movie => movie.imdbID==this.props.id)? check={fav_on:true}:
```

Comment:
Better still, since we use `check` in `cx`, we can simplify above to `const favOn=this.props.favList.find(movie => movie.imdbID==this.props.id);` and then use it as
`const classes=cx('material-icons md-24',{fav_on: favOn});` in cx